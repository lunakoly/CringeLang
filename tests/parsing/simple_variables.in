var a = 10
var b = 20

var c = a * b + 10

let pi, e = 3.14, 2.7

let name = "Nick"