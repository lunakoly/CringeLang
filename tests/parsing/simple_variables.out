==== Raw AST ====
[var [a]: <!MISSING TYPE!><!> = [10], var [b]: <!MISSING TYPE!><!> = [20], var [c]: <!MISSING TYPE!><!> = [((a * b) + 10)], let [pi, e]: <!MISSING TYPE!><!> = [3.14, 2.7], let [name]: <!MISSING TYPE!><!> = ["Nick"]]

==== Resolved AST ====
[var [a]: Int = [10], var [b]: Int = [20], var [c]: [BINARY] = [((a * b) + 10)], let [pi, e]: Real = [3.14, 2.7], let [name]: String = ["Nick"]]

==== Diagnostics ====

==== Done ====
